docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -305,39 +287,35 @@ spec:
   - openstack-networker
 EOF
 ----
-
-Note: alternatively, you could include the `Networker` specific node set in the
-`nodeSets` list when creating the main, `openstack`,
-`OpenStackDataPlaneDeployment` CR.
-
-Note that once a `OpenStackDataPlaneDeployment` CR is deployed, it's impossible
-to add new node sets to it later.
++
+[NOTE]
+Alternatively, you can include the `Networker` specific node set in the
+`nodeSets` list when you create the main `openstack` `OpenStackDataPlaneDeployment` CR, before you deploy it. Note that once an `OpenStackDataPlaneDeployment` CR is deployed, you cannot add new node sets to it later.
```

Comment:
Does this procedure assume (or is it likely) that the original OpenStackDataPlaneDeployment CR is already deployed? And would the main CR necessarily be called `openstack`?

Reply:
CR is not necessarily called `openstack`

Reply:
I would remove `openstack` and just say "main OpenStackDataPlaneDeployment CR."

Reply:
+1